superset-frontend/src/components/Timer.tsx

```diff
@@ -30,8 +30,8 @@ interface TimerProps {
 }
 
 const TimerLabel = styled(Label)`
-  width: 80px;
-  text-align: right;
+  width: 96px;
```

Comment:
If this used the fixed-width / tabular-numerals variant if Inter, we could probably just as easily specify the width in  'em' units, but this seems like it'd work just fine for now, and I'll try to remember to open a PR to address these tabular numerals in the Theme.

Reply:
oh right, `em` seems like the way to go here!